ddtrace/profiling/profiler.py

```diff
@@ -170,6 +170,9 @@ def _build_default_exporters(self):
         if self._lambda_function_name is not None:
             self.tags.update({"functionname": self._lambda_function_name})
 
+        profiling_span_processor = self.tracer._profiling_span_processor
```

Comment:
I'm not a big fan of such tight coupling between components, but I don't know if we could do otherwise at this stage

Reply:
Unfortunately I do not know any other way to do this at this point - we are inevitably collecting some trace information into profile so some coupling will happen.